cds/modules/webhooks/tasks.py

```diff
@@ -73,7 +73,7 @@ def __call__(self, *args, **kwargs):
                 the channel is extracted from the ``sse_channel`` keyword
                 argument.
             """
-            self.sse_channel = kwargs.pop('sse_channel', None)
+            self.sse_channel = kwargs.pop('sse_channel')
```

Comment:
Remove only on `get`, not `pop` (they have different semantics for non-existence).